src/CMakeLists.txt

```diff
@@ -338,6 +338,7 @@ set(SOURCE_FILES
     SlidingWindow.cpp
     Solve.cpp
     SpirvIR.cpp
+    SpirvIR.h
```

Comment:
Why is this a source file and not a header file?-

Reply:
Because we specifically do not want to include it in the set of headers that becomes `Halide.h` as this header includes the upstream SpirvIR headers.

Reply:
Gotcha, may I suggest adding a comment so this doesn't get "fixed" in the future

Reply:
There will be a comment in the PR following the GenGen one; this PR is just meant to minimize future diffs.